Listener/RestListener.php

```diff
@@ -70,7 +70,11 @@ public function onKernelRequest(GetResponseEvent $event)
             } else {
                 $preferredLanguage = $request->getPreferredLanguage($this->locales);
 
-                if ($preferredLanguage !== null) {
+                if (
+                    $preferredLanguage !== null
+                    && count($this->locales) > 0
+                    && in_array($preferredLanguage, $request->getLanguages(), true)
```

Comment:
What's the use-case here? How can it be that we get the locale from request, but it's not in it's languages? Is this the case when we're providing en_US? Can we add a test that the intent would be seen for this part?